Food-Recipe/styles.css

```diff
@@ -267,14 +267,29 @@ button:hover {
   width: 95%;
   max-height: 90%;
   overflow-y: auto;
-
   border-radius: 1em;
   padding: 1em 2em 2em 2em;
   border: 0.2em solid var(--clr-black);
   background-color: var(--clr-white);
   z-index: 2;
 }
 
+.modal::-webkit-scrollbar {
+  width: 10px;
+}
+
+.modal::-webkit-scrollbar-track {
+  background: transparent;
+}
+
+.modal::-webkit-scrollbar-thumb {
+  background-color: #b1b1b1;
+  border-radius: 20px;
+}
+.modal::-webkit-scrollbar-button {
```

Comment:
I added another rule to effectively add padding to the to and bottom. No overflow across all 3 browsers, but still some overflow on mobile in chrome
![image](https://github.com/chingu-voyages/v46-tier1-team-01/assets/113696878/40ca0dc9-0532-4489-8731-1eae80e32a1c)